jira_api.py

```diff
@@ -59,7 +54,14 @@ def add_worklog(self, task_code, date_started, time_spent, comments):
 
 def format_notes(entry_notes):
     notes_row = entry_notes.split('\n')
-    return notes_row[1:len(notes_row)] if len(notes_row) > 1 else ""
+    notes = []
+    for row in notes_row:
+        note = {
+            "text": row,
+            "type": "text"
+        }
+        notes.append(note)
```

Comment:
What do you think to replace these lines to a list comprehension like this?
`[{'text': row, 'type': 'text'} for row in notes_row]`